packages/frontend/app/styles/components/sidebar.css

```diff
@@ -0,0 +1,19 @@
+div.contact-grid {
+    display: grid;
+    grid-template-areas: 'contact pin';
+}
+
+a.contact-display{
+    grid-area: contact;
+}   
+
+button.pin-button{
+    background-color: transparent; 
+    border: none;
+    grid-area: pin;
+    text-align: right;
+}
+
+.pin-icon{
```

Comment:
what color is the icon otherwise?

Reply:
White so it doesn't show

Reply:
found out -- it's the purple background white text :+1: